BSP/Simulator/Src/BSP_PLL.c

```diff
@@ -33,3 +33,31 @@ uint32_t BSP_PLL_GetSystemClock(void) {
     }
     return SystemCoreClock;
 }
+
+/**
+ * @brief   Delays in param micro seconds
+ * @param   # of microseconds to delay
+ * @return  None
+*/
+void BSP_PLL_DelayU(uint32_t micro)
+{
+    uint32_t delay = BSP_PLL_GetSystemClock() / 1000000;
```

Comment:
delay in sim too?

Reply:
I don't think it hurts. If we need it later we can use it. For now, that function doesn't even get called in the sim.